src/components/ValueTypeDropDown.vue

```diff
@@ -4,20 +4,12 @@
 			class="querybuilder__dropdown-label">
 			Value Type
 		</label>
-		<select
+		<Dropdown
 			class="querybuilder__dropdown-select"
-			v-model="selected"
+			v-model="selectedItem"
```

Comment:
No, we can make all that much simpler now. We basically just listen to the `input` event and re-emit a cleaned version that contains only the value we care about. Conversely, we can set the `value` prop here directly.

Then we can also drop all the `watch` methods and the entire local state. We would then only have something like two computed methods and an event listener left

Reply:
I wrestled with it for hours before making this PR, I admit I don't know vue well enough to do what you mean and make it work. Can you try it?

Reply:
Will do!